src/instructlab/model/evaluate.py

```diff
@@ -862,5 +862,5 @@ def evaluate(
                 Benchmark.MMLU_BRANCH, improvements, regressions, no_changes
             )
     except EvalError as ee:
-        print(ee.message)
+        logger.info(msg=ee.message, exc_info=True)
```

Comment:
Can we move this to logger.debug?  All of the messages that come out of EvalError are known conditions and they are caught here to be able to give a user friendly message.  Printing a stack should ideally only be done when the condition wasn't known.

Reply:
Agree, @alimaredia think you missed the above convo on this

Reply:
Practically this would have the effect of printing a stack trace with the user friendly message for lots of [error conditions](https://github.com/instructlab/eval/blob/main/src/instructlab/eval/exceptions.py).  Even in the case that triggered this change, we could tell where they error was being thrown from based on the logging, but if we couldn't I would rather add another exception to differentiate between two cases rather than always print stack traces here.  Essentially every time I see a stack trace, I take it to mean an unhandled exception.  Which isn't the case here.